frappe/hooks.py

```diff
@@ -432,3 +432,7 @@
 extend_bootinfo = [
 	"frappe.utils.telemetry.add_bootinfo",
 ]
+
+naming_series_variables = {
+	"PM": "frappe.tests.test_naming.parse_naming_series_variable",
+}
```

Comment:
Don't add test code in prod code like this. This will run in prod too if someone adds `PM` in naming series... which could've been just text "PM" between dots.

Reply:
Patch hooks in tests:

https://github.com/frappe/frappe/blob/d17a336249713e9663fff7150777196943c27756/frappe/tests/test_website.py#L78-L96